app/models/ability.rb

```diff
@@ -15,16 +15,16 @@ def initialize(user)
 # Leaking personal information!
 #    can :read, CafeWork
     can :main, Rent
+    can [:new, :read, :create], Faq
 
     # Abilities all signed in users get
     if user.id
       can [:main, :new, :edit, :create, :update, :destroy], Rent, user_id: user.id
       can [:edit, :update, :show, :update_password, :update_account], User, id: user.id
       # TODO Implement add_worker
-      # can :add_worker, CafeWork, user_id: nil
-      # can :update_worker, CafeWork, user_id: user.id
-      # can :update_worker, CafeWork
-      # can :remove_worker, CafeWork, user_id: user.id
+      can :add_worker, CafeWork, user_id: nil
+      can :update_worker, CafeWork, user_id: user.id
+      can [:edit, :remove_worker], CafeWork, user_id: user.id
```

Comment:
These should all be on the same line. `can [:edit, :remove_worker, add_worker, update_worker], CafeWork, user_id: user.id`